src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -21,26 +16,17 @@
 using System.Runtime.InteropServices;
 using System.Text;
 
-#if ES_BUILD_STANDALONE
-using System.Runtime.CompilerServices;
-namespace Microsoft.Diagnostics.Tracing
-#else
 using System.Runtime.CompilerServices;
```

Comment:
>using System.Runtime.CompilerServices [](http://example.com/codeflow?start=0&length=37)

This should stay?

Reply:
That using statement was included once inside #if branch and a 2nd time in the #else branch. Eliminating the #if means it is now included once unconditionally. That looks correct to me, am I overlooking an issue?

Reply:
no, I am the one who was overlooking here :-)